e2e_tests/src/_utils/reporter.js

```diff
@@ -0,0 +1,14 @@
+const logger = require('./logger');
+
+class MyCustomReporter {
+  constructor(globalConfig, options) {
+    this._globalConfig = globalConfig;
+    this._options = options;
+  }
+
+  onRunComplete(contexts, results) {
+    logger.log()
+  }
+}
+
+module.exports = MyCustomReporter;
```

Comment:
It is an excellent job @ali-fs :+1: 
LGTM, just one question: this class is defined but never used. is this for the future?

Reply:
Thanks @mahdiaryayi-fs 
This class is used [here](https://github.com/ali-fs/deriv-app/blob/657f61b1070b79588d0734000b2013a536dfb0e4/e2e_tests/jest.config.json#L23) to log the results after the run completed.